tests/unit/py_proxy/views_test.py

```diff
@@ -3,10 +3,26 @@
 import httpretty
 import pytest
 from bs4 import BeautifulSoup
+from h_matchers import Any
 from jinja2 import Environment, FileSystemLoader
 from pkg_resources import resource_filename
+from webtest import TestApp as WebTestApp  # No pytest, this isn't a test class
 
 from py_proxy import views
+from py_proxy.app import app
+
+
+def assert_cache_control(headers, max_age, stale_while_revalidate):
+    # pylint: disable=no-value-for-parameter
+    # Pylint doesn't seem to understand h_matchers here for some reason
+    assert dict(headers) == Any.dict.containing({"Cache-Control": Any.string()})
+
+    header = headers["Cache-Control"]
+    parts = sorted(header.split(", "))
+
+    assert "public" in parts
+    assert f"max-age={max_age}" in parts
+    assert f"stale-while-revalidate={stale_while_revalidate}" in parts
```

Comment:
This feels like it could itself be a matcher: `assert response == Any.response.with.headers(max_age=..., stale_while_revalidate=...)`

Reply:
They aren't actually headers, they are values in the header. It's like this:

`Cache-Control: public, max-age=60, stale-while-revalidate=4853854`

But yes, I'm sure we could put something together